### PR TITLE
fix: fix inconsistent use of the word favourite between US/UK version

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

I believe UK english is prefered in the app to US English.

In any case, we were using inconsistently the word favourite

image

This PR correct a few instances of this.

EDIT: After @marshymarsh comment, I did the reverse, and now I used US name for all. It turns out there's way more changes.

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]

File: libs/common-const/src/bff.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const BFF_BASE_URL = process.env.REACT_APP_BFF_BASE_URL || 'https://bff.cow.fi'
\ No newline at end of file
+export const BFF_BASE_URL = 'http://localhost:3010' // process.env.REACT_APP_BFF_BASE_URL || 'https://bff.cow.fi'
```
Comment (shoom3301) [inline libs/common-const/src/bff.ts:1]: This should be reverted I guess

**Review by alfetopito — COMMENTED** [review]
We don't have to, and I actually vote for not changing component/file names. At a minimum, we should keep localStorage keys to avoid disruption to users.
I know, consistency, but that's not user facing so it doesn't matter IMO.

**Review by shoom3301 — APPROVED** [review]

File: libs/tokens/src/state/tokens/favoriteTokensAtom.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import { DEFAULT_FAVORITE_TOKENS } from '../../const/defaultFavoriteTokens'
+import { TokensMap } from '../../types'
+import { environmentAtom } from '../environmentAtom'
+
+
+export const favoriteTokensAtom = atomWithStorage<Record<SupportedChainId, TokensMap>>(
+  'favouriteTokensAtom:v1',
+  DEFAULT_FAVORITE_TOKENS,
+  getJotaiMergerStorage()
+)
```
[the remark this answers is not in the document]
Comment (anxolin) [reply libs/tokens/src/state/tokens/favoriteTokensAtom.ts:17]: This is not changing, jsut the var name. I kept the old name for the local storage property to not deal with migrations

**Comment by anxolin** [comment]
> We don't have to, and I actually vote for not changing component/file names. At a minimum, we should keep localStorage keys to avoid disruption to users.
I know, consistency, but that's not user facing so it doesn't matter IMO.

Thats what I did in this PR. The only occurrence of the UK term is the local storage name. The exported atom has been renamed though

**Review by alfetopito — APPROVED** [review]
Thanks for clarifying, I did not notice the localStorage key was kept the same.

**Pull request merged**